.github/workflows/.reusable-docker-build.yml

```diff
@@ -67,18 +76,38 @@ jobs:
             type=ref,event=branch
             type=ref,event=tag
             type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
 
       - name: Build and push image
         uses: depot/build-push-action@v1
         with:
+          context: .
           push: true
-          build-args: ${{ inputs.build-args }}
+          platforms: linux/amd64,linux/arm64
```

Comment:
Hmm... do we always want to build for both platforms? Maybe we do.

Reply:
We ship both platforms and this workflow is intended for building our shippable targets.

Reply:
We don't currently do that for _all_ images though I didn't think. But I think we probably should.